libraries/app/config_util.cpp

```diff
@@ -283,6 +283,9 @@ static void create_new_config_file(const fc::path& config_ini_path, const fc::pa
       out_cfg << "\n";
    }
 
+   out_cfg << "\n"
+           << "# Logging configuration is in logging.ini by default.\n"
+           << "# Adding logging configuration here then logging.ini will be ignored, and may cause unexpected issues.\n";
```

Comment:
If logging.ini does not exist, It will very soon, as create_new_logging_file() is called after create_new_config_file().

I believe the logic is such that if logging.ini exists, any logging information here is ignored. So perhaps something like:
``# Adding logging configuration here will have no effect.\n``

Reply:
My bad. Corrected. Thanks.